helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -302,6 +302,64 @@ processors:
      - k8s.namespace.name
      - k8s.pod.name
      - k8s.pod.uid
+  {{- else }}
```

Comment:
Do we need this `else` here? fluentd logs have the same otel attribute names in this chart, so I think it make sense to apply the SCK conventions uniformly